modules/vpn/main.tf

```diff
@@ -75,7 +75,7 @@ resource "aws_eip_association" "vpn" {
 
 // BUCKET AND IAM
 resource "aws_s3_bucket" "furyagent" {
-  bucket_prefix = "${var.name}-vpn-bucket-"
+  bucket_prefix = "${var.name}-${var.vpn_bucket_name_prefix}-"
```

Comment:
this variable name is misleading, as it makes someone think it's the whole prefix, while it's the prefix' suffix.
m2c: if this variable is set, it should be used directly, otherwise `${var.name}-vpn-` should be used.

Reply:
I would add a note, description or label on the resource in case the name is non-conformant to remind the future user of the exception.